guides/v2.3/graphql/reference/store-config.md

```diff
@@ -80,6 +80,22 @@ Attribute |  Data Type | Description
 `no_route` | String | Contains the URL of the default page that you want to appear when if a 404 “Page not Found” error occurs
 `show_cms_breadcrumbs` | Int | Determines if a breadcrumb trail appears on all CMS pages in the catalog. Options: `0` (No) or `1` (Yes)
 
+### Supported Catalog attributes
+
+Use the `catalog` attributes to retrieve information about the store's catalog. These attributes are defined in the `CatalogGraphQl` module.
+
```

Comment:
General comments about this table.

* Add a fourth column to the header and separator rows (lines 87-88). The fourth column should be labeled "Example".
* Alphabetize the attributes